sklearn/preprocessing/label.py

```diff
@@ -304,18 +304,41 @@ def fit(self, y):
         self.classes_ = unique_labels(y)
         return self
 
+    def fit_transform(self, y):
+        """Fit label binarizer and transform multi-class labels to binary
+        labels.
+
+        The output of transform is sometimes referred to by some authors as
+        the 1-of-K coding scheme.
+
+        Parameters
+        ----------
+        y : array or sparse matrix of shape (n_samples,) or \
+(n_samples, n_classes)
+            Target values. The 2-d matrix should only contain 0 and 1,
+            represents multilabel classification. Sparse matrix can be
+            CSR, CSC, COO, DOK, or LIL.
+
+        Returns
+        -------
+        Y : array or CSR matrix of shape [n_samples, n_classes]
```

Comment:
I standardized my usage to always describe shapes using brackets. There was one other place (the `fit` function) in LabelBinarizer that used parens in a similar fashion, so I changed it to use brackets as well.